apps/api/src/app/integrations/usecases/get-active-integration/get-active-integration.command.ts

```diff
@@ -1,3 +1,8 @@
+import { IsBoolean, IsOptional } from 'class-validator';
 import { EnvironmentWithUserCommand } from '../../../shared/commands/project.command';
 
-export class GetActiveIntegrationsCommand extends EnvironmentWithUserCommand {}
+export class GetActiveIntegrationsCommand extends EnvironmentWithUserCommand {
+  @IsBoolean()
+  @IsOptional()
+  filterByEnvironment?: boolean;
```

Comment:
@LetItRock Should we revert this addition? it is not used in this PR but can be a little nice feature to have

Reply:
I would say let's remove it, less code, less thinking, maintaining etc